A/autossh/build_tarballs.jl

```diff
@@ -7,32 +7,59 @@ version = v"1.4.0" # autossh v1.4f, adapted to the BinaryBuilder requirements
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://github.com/Autossh/autossh/releases/download/v1.4f/autossh-1.4f.tgz", "0172e5e1bea40c642e0ef025334be3aadd4ff3b4d62c0b177ed88a8384e2f8f2")
+    ArchiveSource("https://github.com/Autossh/autossh/releases/download/v1.4f/autossh-1.4f.tgz",
+                  "0172e5e1bea40c642e0ef025334be3aadd4ff3b4d62c0b177ed88a8384e2f8f2"),
 ]
 
 # Bash recipe for building across all platforms
+#=
+The ac_cv_func_malloc_0_nonnull and ac_cv_func_realloc_0_nonnull is needed
+to skipping checking malloc for cross-compilation on aarch64-apple-darwin.
+https://github.com/openvenues/libpostal/issues/134
+
+The ac_cv_path_ssh is used to skip the compile-time check on ssh path, to
+avoid hard-coding the absolute path (which is /workspace/destdir/bin/ssh)
+into the autossh binary. With this, the autossh will always use ssh command
+(by searching the PATH variable) in the runtime machine.
+This means we prioritize system ssh executable.
+If needed, the user can still install OpenSSH_jll and use the AUTOSSH_PATH
+variable to change the path of ssh during runtime.
+=#
 script = raw"""
 cd $WORKSPACE/srcdir/autossh*
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --with-ssh=${prefix}/bin \
-            ac_cv_func_malloc_0_nonnull=yes ac_cv_func_realloc_0_nonnull=yes
+
+if [[ "${target}" == *bsd* ]]; then
+    # the configure couldn't correctly check this on BSD
+    conf_args="ac_cv_have_decl___progname=yes"
+else
+    conf_args=""
+fi
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} \
+    ac_cv_func_malloc_0_nonnull=yes ac_cv_func_realloc_0_nonnull=yes \
+    ac_cv_path_ssh=ssh \
+    $conf_args
+
 make -j${nproc}
 make install
+
+cat > ${WORKSPACE}/srcdir/LICENSE << EOF
+Copyright (c) Carson Harding, 2002-2008.
+All rights reserved.
+EOF
+install_license ${WORKSPACE}/srcdir/LICENSE
```

Comment:
Uhm, that's not quite enough of a license, that's just saying all rights are reserved.  Also, years are wrong?  I'm looking at https://github.com/Autossh/autossh/blob/90a8c2f0129f6fe19ec26c7d0fdbab4bb468f476/autossh.c#L7-L22

Reply:
There is a licence in the beginning of the `autossh.c` source file. Looks MIT-ish.

Reply:
In archlinux it's classified as "custom": https://archlinux.org/packages/extra/x86_64/autossh/.  I haven't analysed the differences.

Reply:
Yes I am taking the header of arch package
https://gitlab.archlinux.org/archlinux/packaging/packages/autossh/-/blob/main/LICENSE?ref_type=heads

Unfortunately, the [original package](https://github.com/Autossh/autossh) has no license file, difficult to tell what it is.

Maybe I take exactly the header of the `autossh.c` file?

Reply:
OK Now I am just dumping the header of `autossh.c` as `LICENSE`